sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -73,6 +73,20 @@ class BigQueryEngineAdapter(InsertOverwriteWithMergeMixin):
             },
         },
         support_coercing_compatible_types=True,
+        parameterized_type_defaults={
+            exp.DataType.build("DECIMAL", dialect=DIALECT).this: [(38, 9), (0,)],
```

Comment:
you don't need to pass in dialect if you're just looking for a data type. just pass in the enum directly so you don't have to call this as well

Reply:
it kind of feels like these constants maybe should go into sqlglot, but perhaps we keep it for now